src/Symfony/Component/Serializer/Tests/DependencyInjection/SerializerPassTest.php

```diff
@@ -77,17 +77,20 @@ public function testServicesAreOrderedAccordingToPriority()
 
     public function testBindSerializerDefaultContext()
     {
+        $context = ['enable_max_depth' => true];
+
         $container = new ContainerBuilder();
         $container->setParameter('kernel.debug', false);
-        $container->register('serializer')->setArguments([null, null]);
-        $container->setParameter('serializer.default_context', ['enable_max_depth' => true]);
+        $container->register('serializer')->setArguments([null, null, null]);
```

Comment:
```suggestion
        $container->register('serializer')->setArguments([null, null]);
```
 To stick with the initial service definition

Reply:
I just saw that, thanks 👍🏽